test/utils/ErrorHandlerSpec.scala

```diff
@@ -67,7 +68,9 @@ class ErrorHandlerSpec extends UnitSpec with GuiceOneAppPerSuite {
     (auditConnector.sendEvent(_ : DataEvent)(_: HeaderCarrier, _: ExecutionContext)).expects(*, *, *)
       .returns(Future.successful(Success))
 
-    val configuration = Configuration("appName" -> "myApp")
+    val configuration = Configuration(
+      "appName" -> "myApp",
+      "bootstrap.errorHandler.warnOnly.statusCodes" -> List.empty)
```

Comment:
What is this for?

Reply:
This was to fix an error within the ErrorHandlerSpec which kept flagging because it couldn't find a configuration setting with the key 'bootstrap' in it.

`com.typesafe.config.ConfigException$Missing: hardcoded value: No configuration setting found for key 'bootstrap'`

This change can also be found in Yang's PR here https://github.com/hmrc/individuals-disclosures-api/pull/39/files